kernel/libs/aster_page_allocator/src/lib.rs

```diff
@@ -0,0 +1,68 @@
+// SPDX-License-Identifier: MPL-2.0
+
+//! This crate provides a global page allocator for the kernel.
+//!
+//! Currently, the page allocator is implemented by the buddy system.
+//! By corporating with the `PageAlloc` trait and `page_allocator_init`
```

Comment:
Still it is important to polish documentations in the published crates. You can refer to the OSTD API in the documentation by such syntax:

```rust
//! ...
//! By corporating with the [`PageAlloc`] trait and the [`page_allocator_init`]
//! procedural macro.
//! ...
//!
//! [`PageAlloc`]: ostd::mm::PageAlloc
//! [`page_allocator_init`]: ostd::page_allocator_init
```